### PR TITLE
[CI] Test Flocq master

### PR DESCRIPTION
**Pull request by proux01**
Following merge of https://github.com/AbsInt/CompCert/pull/368

### CODE REVIEW

File: dev/ci/ci-basic-overlay.sh [hunk 1]
```diff
@@ -107,7 +107,12 @@ project geocoq "https://github.com/GeoCoq/GeoCoq" "master"
 ########################################################################
 # Flocq
 ########################################################################
-project flocq "https://gitlab.inria.fr/flocq/flocq" "flocq-3"
+project flocq3 "https://gitlab.inria.fr/flocq/flocq" "flocq-3"
```
Comment (ppedrot) [inline dev/ci/ci-basic-overlay.sh:110]: You also have to change the header name above.

File: dev/ci/ci-flocq3.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download flocq
```
Comment (silene) [inline dev/ci/ci-flocq3.sh:8]: `flocq3`?

File: dev/ci/ci-flocq3.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download flocq3
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/flocq"
```
[the remark this answers is not in the document]
Comment (proux01) [reply dev/ci/ci-flocq3.sh]: Damn

**Comment by proux01** [comment]
Compcert and VST are now green.
Th Windows failure is unrelated and will be fixed by https://github.com/coq/coq/pull/15967

**Comment by ppedrot** [comment]
Let's mark this as ready then.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**